src/plugins/elasticsearch/ExplorerNG/Main/Raw/index.tsx

```
@@ -88,7 +88,7 @@ interface Props {
 
 export default function index(props: Props) {
   const { t } = useTranslation(NAME_SPACE);
```

Biome (2.4.4)

[error] This hook is being called from within a function or method that is not a hook or component.

(lint/correctness/useHookAtTopLevel)